wled00/FX.cpp

```diff
@@ -2042,7 +2071,7 @@ uint16_t mode_palette() {
   }
   return FRAMETIME;
 }
-static const char _data_FX_MODE_PALETTE[] PROGMEM = "Palette@Shift,Size,Rotation,,,Animate Shift,Animate Rotation,Anamorphic;;!;12;c1=128,c2=128,c3=128,o1=1,o2=1,o3=0";
```

Comment:
The rotation was put there intentionally by last modifier (@TripleWhy) and I agreed to that.
I do understand that it makes less sense for a single, undivided strip but I would like to avoid yet another breaking change.

Reply:
There are plenty of complaints I receive that say, in one way or the other, this effect changed when users click on UI elements. This is one of those situations.

But yes, you are correct, 0_15 is not yet released so anything can happen. I was just worried that you might get frustrated when your change was modified. If you are not, I am ok with the change as I normally don't use rotation on 1D strips.

Reply:
I am ok with either, it was just that I have seen user reports about this FX being 'broken' on 1D as they did not notice the rotation button.